src/AbstractPluginManager.php

```diff
@@ -301,9 +299,7 @@ public function addAbstractFactory(string|AbstractFactoryInterface $factory): vo
      *
      * @param string $name Service name
      * @param string|callable|DelegatorFactoryInterface $factory Delegator factory to assign.
-     * @phpcs:disable Generic.Files.LineLength.TooLong
-     * @psalm-param class-string<DelegatorFactoryInterface>|class-string<object&DelegatorCallable>|DelegatorCallable $factory
-     * @phpcs:enable Generic.Files.LineLength.TooLong
+     * @psalm-param class-string<DelegatorFactoryInterface>|DelegatorCallable $factory
```

Comment:
Using intersections with callable in a class string parameter is not supported, i.e. `class-string<object&callable>` causes an `InvalidDocblock` issue in Psalm